docfx_project/ziti/clients/linux.md

```diff
@@ -8,7 +8,51 @@ The purpose of the tunneller is to configure host access. This means all users a
 
 ### Installation and Upgrade
 
-[The latest release](https://github.com/openziti/ziti-tunnel-sdk-c/releases/latest/) of `ziti-edge-tunnel` is distributed as a binary executable from GitHub. The upgrade procedure is identical to the installation procedure.
+#### RPM
+
+To install the x86_64 RPM you first create a repo file like `/etc/yum.repos.d/openziti.repo` resembling the examples below. Then, you run `yum update` to refresh your repodata cache so you can run `yum install ziti-edge-tunnel` to install the RPM. To upgrade you run `yum update ziti-edge-tunnel`.

```

Comment:
Does it only 'resemble' the examples below? Can we not be authoritative and state exactly like the example below? I would rather prescribe to people 'how to' and be authoritative. Is that not possible?

I try to not use "you" in this doc - instead reword it to say "Then run" or "once in place, issue this command". I think this could be bullet points too?